packages/uikit/createLinks.sh

```diff
@@ -0,0 +1,9 @@
+dependencies=( "react" "react-dom" "react-icons" )
```

Comment:
Move this to a folder `/scripts` and create a npm script, like `link-deps`.
Anyway, this PR removes the `react-icons` dependency.